### ISSUE TITLE
Unable to import rayoptics

### ISSUE THREAD

**Issue by dibyendumajumdar** [mjhoptics/ray-optics#4]
Hi,

I am trying out your cool project. I installed in on my Linux machine (Redhat 7.7). 
Installation was successful but if I try:

```
from rayoptics.environment import *
```

I get following error:
```
>>> from rayoptics.environment import *
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dylan/pydev/lib64/python3.6/site-packages/rayoptics/environment.py", line 26, in <module>
    from rayoptics.gui.appcmds import create_new_model
  File "/home/dylan/pydev/lib64/python3.6/site-packages/rayoptics/gui/appcmds.py", line 14, in <module>
    from rayoptics.mpl.axisarrayfigure import Fit
  File "/home/dylan/pydev/lib64/python3.6/site-packages/rayoptics/mpl/axisarrayfigure.py", line 14, in <module>
    from scipy.interpolate import spline
ImportError: cannot import name 'spline'

```

The installed version of scipy is 1.3.1.
There doesn't seem to be a 'spline' attribute in scipy.interpolate.

I am not sure what the issue is - I suspect your project is not compatible with scipy 1.3.1?
Please let me know if there is a workaround.

Thanks and Regards
Dibyendu

**Comment by dibyendumajumdar** [comment]
I tried on Windows 10 with Python 3.7. Got the same error unfortunately.

**Comment by bergkoet** [comment]
The scipy docs say `scipy.interpolate.spline` was removed in version 1.3.0:
https://docs.scipy.org/doc/scipy/reference/release.1.3.0.html#scipy-interpolate-changes

The deprecation warning says to use `Bspline` instead:
https://docs.scipy.org/doc/scipy-1.2.1/reference/generated/scipy.interpolate.spline.html#scipy.interpolate.spline

**Comment by mjhoptics** [comment]
Sorry, I was asleep at the wheel and didn't see your issue. I updated a fix for this on master, commit f9f660c5fdaacb31707132f00257dbae51ecd880 on 9/24/19.

**Comment by mjhoptics** [comment]
This should be resolved in release 0.4.6 on pypi. The final git commit is 370259b6c0eea773dd8c51c114306764eb21af72
